### DISCUSSION TITLE
Please support nixos

### DISCUSSION THREAD

**Discussion by OBJ0928** [end-4/dots-hyprland#128] [Ideas]
the dotfiles are cool but just don't work on nixos

**Comment by end-4** [comment]
i will not use nixos due to my laziness
now, i don't know how nixos works so i can't say for sure, but i don't think not providing a "flake" or something like that would restrict users of the funny flake distro from using it

**Comment by end-4** [comment]
okay im trying to use nixos hopefully i can get a flake ready soon lol

**Reply by solitudealma** [reply]
> okay im trying to use nixos hopefully i can get a flake ready soon lol好吧，我正在尝试使用 nixos，希望我能尽快准备好薄片，哈哈

can you list extraPackages,because i am not sure which additional packages need to be installed

**Reply by ghost** [reply]
no freaking way I can finally be at peace knowing my AGS config can finally satisfy me

**Reply by sitolam** [reply]
Any progress on nixos?

**Reply by end-4** [reply]
not happening unless i SOMEHOW become a sysadmin and learn nixos

**Reply by sitolam** [reply]
Is there not somebody else here with enough nixos skills that you can work together with? This repo justs looks perfect for nixos

**Comment by rbutera** [comment]
@end-4 thank you thank you thank you for trying nixos, your dotfiles are the only reason I am still on arch. I want to move to nixos really badly!

**Reply by solitudealma** [reply]
you can use his anotherrepo for nixos. add one package for font can use it

**Reply by rbutera** [reply]
does it have all his ags widgets?

**Reply by solitudealma** [reply]
yes， i am using

**Reply by end-4** [reply]
You mean CirnOS? Keep in mind that the ags config in that repo is old and I never actually got to a point where I'm totally comfortable with it.

**Reply by solitudealma** [reply]
> You mean CirnOS? Keep in mind that the ags config in that repo is old and I never actually got to a point where I'm totally comfortable with it.

Although it is an old configuration, it can be used as long as a font package is added. Then a problem that arises is the problem that the volume icon I mentioned is not displayed

**Comment by jee-mj** [comment]
[This is a full configuration for a system but you can get by with `homemanager/modules`](https://github.com/sammhansen/ags-nix)
